lib/index.js

```diff
@@ -102,6 +125,24 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
 
   app.options(accountPath, cors());
 
+  app.get(
+    `${accountPath}/roles`,
+    ensureAuthenticated,
+    cors(),
+    asyncHandler(async (req, res) => {
+      /**
+       * @func getCapabilitiesForAnAccount
+       * @param {Request} req
+       * @param {Response} res
+       * @description uses the account param to get
+       * all the capabilities for an account
+       * @route /:account/caps
```

Comment:
Docs need to be updated to say "roles" instead of "caps".

Reply:
great castch